### DISCUSSION TITLE
Option to change behavior of smart-tab inside comments

### DISCUSSION THREAD

**Discussion by TUSF** [helix-editor/helix#12110] [Ideas]
Currently, if I have something like so:

```
// TODO:    Some aligned todo comment.
//<cursor>
```
And decide to use `tab` to indent the line inside the comment, the `<cursor>` ends up jumping to  end of the file (or the next node).

I think there should be an option for smart-tab to treat the inside of a comment the same way it treats whitespace, and so in the above case, it inserts the tab/space characters accordingly.

**Comment by masriomarm** [comment]
I suggest to swap smart-tab keybinding 
So, `s-tab` should trigger `smart-tab` and `tab` is just tab

So, those who want to trigger smart-tab press the extra binding.

**Comment by ghuh** [comment]
I'm having this same issue and it is super annoying.  I agree, there needs to be some way to change the behavior.
